### DISCUSSION TITLE
Soroban Quest 1-2 Videos

### DISCUSSION THREAD

**Discussion by rawritude** [stellar/sorobanathon#44] [General]
Hey Stellar peeps!
Started a video series on the new Stellar Quest Live Series 5 (AKA Soroban Quest). In each video I go over an explanation of the task at hand, readings parts of the contract and completing the quest using the CLI.

### Stellar Soroban Quest Live 1 - Hello World (Build, Deploy, Invoke)
In the first video I explain how to build the Hello World contract, deploy it to Futurenet, and invoke it.
-->[Link to video](https://www.youtube.com/watch?v=XcFgR_OHKl8)

### Stellar Soroban Quest Live 2 - Auth Store (Store and Retrieve)
In the second video, I explain how to store and retrieve data in from a deployed smart contract.
[-->Link to video](https://www.youtube.com/watch?v=kpyIotaiFfw)

![Soroban Quest 2](https://user-images.githubusercontent.com/7675978/202090213-e107f276-7542-4e28-b9ca-2fc271f70e22.png)

**Comment by achallagundla** [comment]
Thank you so much for these awesome videos! 
Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission qualifies for **a reward of 6000 XLM (3000 per video)**!

One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.
As always lmk if you have any questions on discord!
